wiiuse/io_wii.c

```diff
@@ -138,12 +141,43 @@ int wiiuse_register(struct wiimote_listen_t *wml, struct bd_addr *bdaddr, struct
 	return 0;
 }	
 
+int wiiuse_connect(struct wiimote_listen_t *wml, struct bd_addr *bdaddr, u8 *name, struct wiimote_t *(*assign_cb)(struct wiimote_listen_t *wml, u8 err))
```

Comment:
this is a problem and weird. in both wiiuse upstream as our fork `wiiuse_connect`'s signature is `int wiiuse_connect(struct wiimote_t** wm, int wiimotes)`. how does it compile this? can we make a connect work with that signature? 

i ask this because switching to upstream means that wiiuse_connect will be the mentioned signature, and pass the info to a OS (in this case libogc) implementation of the connect function. 
the wiimote_t should have all the info you pass here afaik. could you check?
wiiuse: https://github.com/wiiuse/wiiuse/blob/master/src/wiiuse.h#L730
current libogc fork : https://github.com/devkitPro/libogc/blob/master/gc/wiiuse/wiiuse.h#L594